CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 - Kotlin 1.9.0 is used; `kotlin-stdlib-jdk8` dependency was replaced with `kotlin-stdlib` due to [Kotlin updates](https://kotlinlang.org/docs/whatsnew18.html#updated-jvm-compilation-target).
 - Allow application to reset the oauth token cached within the server, when it deems that it is no longer valid
+- Corrected cardinality and range of the oslc_config:acceptedBy property
```

Comment:
please specify from what to what, eg

> from `String[0..1]` to `Resource[0..*]`

Reply:
also, if it's not a fix for a regression, I think it would make to prefix this with an emoji 🧨 that I usually use for breaking changes.